tutorials/alcf/02_ThetaGPU_ray/README.rst

```diff
@@ -54,6 +54,7 @@ Add the following content:
     #COBALT -n 2
     #COBALT -q full-node
     #COBALT -t 20
+    #COBALT --attrs filesystems=home,grand,eagle,theta-fs0
```

Comment:
add a note/tip about this, say that deephyper only requires `theta-fs0, home` + whatever directory is used by the user